.vscode/tasks.json

```diff
@@ -15,6 +15,20 @@
       },
       "problemMatcher": []
     },
+    {
+      "label": "Preview SkipPip",
```

Comment:
Should this be in this PR? If this is something that would be handy, would be best to have it in a separate PR.

Reply:
already removed